spec/features/events_spec.rb

```diff
@@ -118,25 +120,30 @@
     end
   end
 
-  scenario "logged in as Organizer I can change application status with radio buttons" do
+  scenario "logged in as Organizer I can change application status with radio buttons in selection phase" do
     login(:organizer)
-    @event.application_status_locked = false
-    @event.save
     @pupil = FactoryGirl.create(:profile)
     @application_letter = FactoryGirl.create(:application_letter, event: @event, user: @pupil.user)
+    @event.application_status_locked = false
```

Comment:
Maybe just creating another event in selection phase here would be more readable?